themes/basic/style.css

```diff
@@ -0,0 +1,3 @@
+/**
+ * Theme Name: basic
```

Comment:
Would this theme be better named "require-login" or something?

Reply:
Yeah that makes more sense! Good shout.

Reply:
```suggestion
 * Theme Name: require-login
```